climatetree/frontendfinal
# Improve Dropdown Suggestions

### Change type:
- New feature

###Description:
- Updates place suggestion dropdown to include the type of place and provide more checks on the contextual information shown.

### Steps to Verify This PR
- Go to map and try searching "new york", "sea", "united states" or any other search term you like as long as you see one of each type of place.
